src/api/axios.ts

```diff
@@ -0,0 +1,76 @@
+import "../config/env.js";
+import axios, { AxiosInstance, AxiosRequestConfig } from "axios";
+import { getApiToken } from "./config.js";
+
+const BASE_URL =
+  process.env.FIREBERRY_API_URL || "https://app.fireberry.com/api/v3";
+
+const apiClient: AxiosInstance = axios.create({
```

Comment:
prefer named exports
```suggestion
export const apiClient: AxiosInstance = axios.create({
```